ci/test.yml

```
@@ -0,0 +1,18 @@
+image: $CI_DEPENDENCY_PROXY_GROUP_IMAGE_PREFIX/nixos/nix:latest
```

```suggestion
image: $CI_DEPENDENCY_PROXY_DIRECT_GROUP_IMAGE_PREFIX/nixos/nix:latest
```
This is related to #132